docs/guides/cloud/debugging/test-replay.mdx

```diff
@@ -3,4 +3,140 @@ title: Test Replay
 sidebar_position: 20
 ---
 
-Test Replay!
+:::info
+
+## <Icon name="graduation-cap" /> What you'll learn
+
+- What Test Replay is and why you should use it
+- How to get started with Test Replay in Cypress Cloud
+- How to debug failed and flaky CI test runs more accurately and efficiently
+
+:::
+
+:::caution
+
+{/* todo: change changelog url to real page when it's ready/won't break build */} <br/>
+&#8239;<Icon name="exclamation-triangle" /> Test Replay is available when recording tests to Cypress Cloud using Cypress `v13` and up. See the [Cypress v13 changelog](/guides/references/changelog#12-0-0).
+
+:::
+
+:::caution
+
+Test Replay currently supports Chromium-based browsers only.
```

Comment:
I wonder if we should elaborate on what browsers are / are not supported – there may be customers who do not know what browsers are Chromium-based.

Reply:
Can't hurt 👍